sponsors.yml

```diff
@@ -63,15 +63,15 @@
   enddate:
 
 - name: Allston Trading
-# logo: no logo designed
-# banner: no available svg
+  # logo: no logo designed
+  # banner: no available svg
   level: silver
   startdate:
   enddate:
- 
+
```

Comment:
This is why your PR was failing, there were a couple spots where white space was left